sprint-mission-7/queries.sql

```diff
@@ -0,0 +1,153 @@
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+UPDATE users SET username = 'test' WHERE id = 1;
+
+/*
+  2. 내가 생성한 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+SELECT *
+FROM products
+WHERE seller_id = 1
+ORDER BY created_at DESC
+LIMIT 10 OFFSET 20;
+
+
+/*
+  3. 내가 생성한 상품의 총 개수
+*/
+SELECT COUNT(*) AS total_products
+FROM products
+WHERE seller_id = 1;
+
+/*
+  4. 내가 좋아요 누른 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+SELECT p.*
+FROM products p
+JOIN likes l ON p.id = l.product_id
+WHERE l.user_id = 1
+ORDER BY l.created_at DESC
+LIMIT 10 OFFSET 20;
+-- 위의 코드와 차이점은?
+SELECT * 
+FROM products
+JOIN likes ON products.id = likes.product_id
+WHERE likes.user_id = 1
+ORDER BY likes.created_at DESC
+LIMIT 10 OFFSET 20;
+
+/*
+  5. 내가 좋아요 누른 상품의 총 개수
+*/
+SELECT 
+	COUNT(*) AS liked_count
+FROM likes
+WHERE user_id = 1;
+
+/*
+  6. 상품 생성
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+INSERT INTO products
+	(seller_id, product_name, description, price, image_url)
+	VALUES
+	(1, '샘플상품', '10글자이상 설명입니다.', 10000, 'https://example.com/sample.jpg')
+;
+
+SELECT * FROM products
+-- UPDATE products SET product_name = '상품F'WHERE id = 6;
+-- UPDATE products SET description = '상품F의 상세 설명입니다.' WHERE id = 6;
+
+/*
+  7. 상품 목록 조회
+  - "test" 로 검색
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 1번째 페이지
+  - 각 상품의 좋아요 개수를 포함해서 조회하기
+*/
+SELECT 
+  products.id,
+  COUNT(likes.user_id) AS like_count,
+  products.product_name,
+  products.description,
+  products.price,
+  products.image_url,
+  products.created_at
+FROM products
+LEFT JOIN likes ON products.id = likes.product_id
+WHERE products.product_name LIKE '%test%'
+GROUP BY 
+  products.id,
+  products.product_name,
+  products.description,
+  products.price,
+  products.image_url,
+  products.created_at
```

Comment:
```suggestion
GROUP BY 
  products.id
```
와 동일한 결과이기 때문에 생략가능할 것으로 보여요